README.md

```diff
@@ -3,54 +3,100 @@
 The `py-allotax` implements a python interface to the `allotaxonometer` library. The tool here provides a way for users to input data and arguments and receive back a saved plot! The tool is designed to be used in a command line or in a python notebook in a few lines of code (see usage instructions at the bottom).
 
 Table of contents:
-- [Installation steps](#installation-steps)
+- [Package Build](#package-build)
 - [Usage instructions](#usage-instructions)
+- [Developer Notes](#developer-notes)
 - [Frequent questions or issues](#frequent-questions-or-issues)
 - [Repo structure notes](#repo-structure-notes)
 - [Resources](#resources)
 
 
+
 ## Installation steps
 
+1. Install `python3.11` or greater.
+
 1. If JavaScript tool installs are needed (never used or installed `npm`, `nvm`, `node`):
```

Comment:
If we use docker, do we still need to talk about JS installation steps? Maybe we could hide that in developer notes.